recipe/build.sh

```diff
@@ -22,9 +24,15 @@ chmod +x configure
             --disable-fortran
 
 make
+
+# ncgen segfaults on macOS
+if [[ $(uname) != Darwin ]]; then
```

Comment:
This is not ideal but I noticed that the tests are skipped in `defaults` too. We should at skip only the failing test though.